core/src/main/java/org/apache/calcite/avatica/BuiltInConnectionProperty.java

```diff
@@ -86,7 +86,10 @@
   KEY_PASSWORD("key_password", Type.STRING, "", false),
 
   HOSTNAME_VERIFICATION("hostname_verification", Type.ENUM, HostnameVerification.STRICT,
-      HostnameVerification.class, false);
+      HostnameVerification.class, false),
+
+  /** The number of rows to fetch per call, default is 100 rows. */
+  FETCH_ROW_COUNT("fetch_row_count", Type.NUMBER, AvaticaStatement.DEFAULT_FETCH_SIZE, false);
```

Comment:
What do you think of `default_fetch_size`?

Reply:
I would second @julianhyde 's  [feedback here](https://issues.apache.org/jira/browse/CALCITE-2322?focusedCommentId=17370941&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17370941):

> not "default_fetch_row_count" (because it is understood that this is a default that can be overridden per-statement; furthermore, there is a default default, which is 100);